### PR TITLE
Flesh out the readme for the application

### PR DESCRIPTION
**Pull request by danielbachhuber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
+See `.env.sample` for expected environment variables.
+
+### Generating OAuth tokens
+
+OAuth tokens are needed to authorize requests to Facebook. They are stored as environment variables as well.
+
+First we need to generate the credentials:
+
+```bash
+$ mtfv oauth facebook_client_secrets.json facebook-creds.json publish_actions
```
Comment (danielbachhuber) [inline README.md]: @mpatek @scribu Do you know where the format of `facebook_client_secrets.json` comes from?

**Comment by danielbachhuber** [comment]
@mpatek @scribu what else do you think is worth clarifying about the readme?

File: README.md (hunk 1 above)
Comment (scribu) [inline README.md]: Not sure if it's a standard. When you generate an OAuth 2.0 client ID in the Google Developer Console, you can download the info as a JSON file and that's the format it comes in.

Comment (danielbachhuber) [inline README.md]: Ok. I've included a sample in the readme.

**Pull request merged**